drivers/ptp_clock/Kconfig.nxp_enet

```diff
@@ -5,5 +5,6 @@ config PTP_CLOCK_NXP_ENET
 	bool "NXP ENET PTP Clock driver"
 	default y
 	depends on DT_HAS_NXP_ENET_PTP_CLOCK_ENABLED && NET_L2_PTP
+	depends on ETH_NXP_ENET
```

Comment:
I'm not familiar with the PTP testcase, but it seems that the `tests/net/ptp/clock/prj.conf` file sets `CONFIG_NET_TEST`, which in turn disables `CONFIG_ETH_DRIVER`. This causes `CONFIG_ETH_NXP_ENET` to be false as well, so the PTP driver won't be built either because of this dependency.

Is this intentional? It seems like we'd want the PTP driver to be built by this testcase.

Reply:
It seems if I add a line like the following:

```diff
--- a/mcux/hal_nxp.cmake
+++ b/mcux/hal_nxp.cmake
@@ -160,6 +160,7 @@ include_driver_ifdef(CONFIG_ENTROPY_MCUX_RNGA               rnga            driver_rnga)
 include_driver_ifdef(CONFIG_ENTROPY_MCUX_TRNG          trng            driver_trng)
 include_driver_ifdef(CONFIG_ENTROPY_MCUX_CAAM          caam            driver_caam)
 include_driver_ifdef(CONFIG_ETH_NXP_ENET               enet            driver_enet)
+include_driver_ifdef(CONFIG_PTP_CLOCK_NXP_ENET         enet            driver_enet)
 include_driver_ifdef(CONFIG_HAS_MCUX_SMC               smc             driver_smc)
 include_driver_ifdef(CONFIG_I2C_MCUX                   i2c             driver_i2c)
 if (CONFIG_NXP_LP_FLEXCOMM)
```
to the MCUX HAL that we can build the PTP driver with this testcase

Reply:
Based on @jukkar's comment it seems we don't want to test the PTP driver's build with this sample, so adding this dependency would make sense

Reply:
yes the point of this change is to make it not build